src/cluecode/copyrights.py

```diff
@@ -2562,6 +2562,9 @@ def build_detection_from_node(
     #holder is Tim Hudson (tjh@mincom.oz.au).
     COPYRIGHT: {<HOLDER> <JUNK> <NAME-EMAIL>}       #83001
 
+    # Copyright lowRISC contributors.
+    COPYRIGHT: {<COPY> <NN> <CONTRIBUTORS>}
+
```

Comment:
That's great! do you mind to add a test for this too?

Reply:
Sure thing :)